### PR TITLE
Apply visual update for /operator-day

### PR DESCRIPTION
**Pull request by mtruj013**
## Done

- Updated copy and styling as per [feedback](https://github.com/canonical-web-and-design/ubuntu.com/issues/11797#issuecomment-1176298126) from when page was originally built
- Copy doc [here](https://docs.google.com/document/d/1ZFRGkZ4vvvMVOgq4MIsBuebHsA7Y0rTyiZ76qjQpOAg/edit#) (note: copy is a copy/paste from each youtube videos description, this was +1'd by pdm)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://juju-is-429.demos.haus/operator-day
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check against linked comment and see that all points were addressed 

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/5756

### CODE REVIEW

File: templates/operator-day/index.html [hunk 1]
```diff
@@ -12,21 +12,28 @@ <h1>Operator Day</h1>
...
   <div class="u-fixed-width u-sv3">
-    <h2>May 2022</h2>
+    <h2 class="p-heading--3">May 2022</h2>
   </div>
 
   <div class="row">
-    <div class="col-8">
+    <div class="col-7">
       <h3 class="p-heading--4"><a href="https://www.youtube.com/watch?v=kHG_nCSv51s">Plenary session with Mark Shuttleworth and David Booth</a></h3>
+      <p>“Kubernetes is taking the app development world by storm.” is one of the most popular phrases about Kubernetes. How can we make best use of this technology?</p>
```
Comment (minkyngkm) [inline templates/operator-day/index.html:27]:
```suggestion
      <p>"Kubernetes is taking the app development world by storm." is one of the most popular phrases about Kubernetes. How can we make best use of this technology?</p>
```

**Review by minkyngkm — DISMISSED** [review]
Looking good @mtruj013 just one thing though! The videos look too small on medium screens. Could we fix this? :)

image

**Comment by Lukewh** [comment]
@minkyngkm do you have any details about why your QA failed?

**Comment by minkyngkm** [comment]
@Lukewh I just thought it would look better if the videos more fits without too much white spaces in the screen size 620px - 1030px! Everything works well though. Just the spaces!

**Comment by mtruj013** [comment]
@Lukewh @minkyngkm it's (in a round about way) because of the thumbnail. If I make the video larger, it looks as it should on medium screens, but on large screens the thumbnail looks cut off (example below)

Screenshot 2022-08-03 at 17 19 30
VS
Screenshot 2022-08-03 at 17 19 38

**Review by sowasred2012 — APPROVED** [review]
LGTM
lgtm

**Pull request merged**